frontend/src/ofrak/remote_resource.js

```diff
@@ -435,19 +463,36 @@ export class RemoteResource extends Resource {
   }
 
   async search_for_vaddr(vaddr_start, vaddr_end) {
-    const matching_models = await fetch(`${this.uri}/search_for_vaddr`, {
-      method: "POST",
+    const matching_models = await fetch(
+      `${backendUrl}/${this.uri}/search_for_vaddr`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify([vaddr_start, vaddr_end]),
+      }
+    ).then(async (r) => {
+      if (!r.ok) {
+        throw Error(JSON.stringify(await r.json(), undefined, 2));
+      }
+      return await r.json();
+    });
+    return remote_models_to_resources(matching_models);
+  }
+
+  async get_script() {
```

Comment:
Perhaps `fetch_script`?

Reply:
I considered that, I didn't suggest it because "fetch" still carries some connotation of returning a value, though less so than "get." I feel "update" is the most clear verb in this case.